test cases/frameworks/38 gettext extractor/src/lib3/meson.build

```diff
@@ -0,0 +1,7 @@
+touch = find_program('touch', required: false, disabler: true)
+
+foo = custom_target('foo',
+  output: 'foo.c',
+  command: [touch, '@OUTPUT@'],
+  )
```

Comment:
I don't know if @jpakkane is okay with using `touch` here, but if not using the `fs.copyfile()` function returns a `custom_target` as is known to work everywhere.